rise-playlist.html

```diff
@@ -176,8 +168,8 @@
           duration = parseInt(duration, 10);
 
           if (!isNaN(duration)) {
-            setTimeout(function() {
-              self._playNext(index, "pause");
+            this.debounce("play-next", function() {
+              this._playNext(index, "pause");
```

Comment:
@donnapep I don't see anywhere where you cancel `"play-next"` in this component, I think `this.async` might be more appropriate here.